style.css

```diff
@@ -1,7 +1,396 @@
+  position: absolute;
+  top: 8px;
+  width: 23px;
+}
+
+.burger::after {
+  background-color: #fff;
+  bottom: 8px;
+  content: " ";
+  height: 3px;
+  margin: 0;
+  position: absolute;
+  width: 23px;
+}
+
+.introduce {
+  color: #e1e6e9;
+  display: flex;
+  flex-direction: column;
+  font-weight: 900;
+  justify-content: flex-end;
+  padding-bottom: 419px;
+}
+
+.introduce__title {
+  align-self: flex-end;
+  font-size: 40px;
+  margin-bottom: 11px;
+  margin-right: 0;
+  margin-top: 0;
+}
+
+.introduce__paragraph {
+  align-self: flex-end;
+  font-size: 20px;
+  margin: 0;
+  margin-bottom: 39px;
+  max-width: 283px;
+  text-align: right;
+}
+
+.button {
+  background-color: #ffda45;
+  border: 0;
+  box-shadow: 5px 5px 20px rgba(0, 0, 0, .25);
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 900;
+  height: 55px;
+  outline: none;
+  text-align: center;
+  width: 200px;
+}
+
+.button_mediumvioletred {
+  align-self: flex-end;
+  background-color: #e0036f;
+  color: #fff;
+}
+
+.tickets {
+  align-items: center;
+  background-color: #e3eafe;
+  display: flex;
+  flex-direction: column;
+  padding-top: 55px;
+  padding-bottom: 55px;
+}
+
+.title {
+  color: #585858;
+  font-weight: 500;
+  font-size: 30px;
+}
+
+.tickets__title {
+  margin: 0;
+  margin-bottom: 40px;
+  text-align: center;
+  max-width: 621px;
+}
+
+.tickets__paragraph {
+  color: #263b83;
+  font-weight: 900;
+  font-size: 30px;
+  margin: 0;
+  margin-bottom: 33px;
+}
+
+.tickets__button {
+  border: 0;
+  box-shadow: 5px 5px 20px rgba(0, 0, 0, .25);
+  margin-bottom: 18px;
+}
+
+.tickets__reserve {
+  background-color: #e3eafe;
+  border-bottom: 3px solid #ffda45;
+  border-left: 0;
+  border-right: 0;
+  border-top: 0;
+  color: #585858;
+  font-size: 20px;
+  font-weight: 900;
+  outline: none;
+  padding-bottom: 13px;
+}
+
+.participants {
+  align-items: center;
+  background-color: #405dbb;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 90px;
+  max-height: 442px;
+  max-width: 1440px;
+}
+
+.wrap-info {
+  color: #fff;
+  font-weight: 900;
+  padding-left: 130px;
+  padding-top: 118px;
+}
+
+.wrap-info__title {
+  font-size: 30px;
+  margin: 0 0 20px 0;
+  max-width: 227px;
+}
+
+.wrap-info__paragraph {
+  font-size: 20px;
+  padding-bottom: 108px;
+  padding-right: 43px;
+  margin: 0;
+  max-width: 430px;
+}
+
+.wrap-info__pic {
+  max-height: 327px;
+  max-width: 464px;
+  padding-right: 43px;
+}
+
+.button__request {
+  margin-right: 131px;
+}
+
+.speakers {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.speakers__title {
+  text-align: center;
+  margin-bottom: 78px;
+  margin-top: 0;
+  max-width: 451px;
+}
+
+@media screen and (max-width: 768px) {
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    padding-bottom: 15px;
+    padding-top: 33px;
+  }
+
+  .nav__link { display: none; }
+
+  .burger { display: block; }
+}
+
+@media screen and (max-width: 320px) {
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)